projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -127,6 +131,7 @@ export class TuiEditorNewComponent
 
     removeLink(): void {
         this.editor?.unsetLink();
+        this.dropdownSelection?.closeDropdownBox();
```

Comment:
Wouldn't that break tuiPure on the toggle? Why is it necessary to manually close the dropdown?

Reply:
reverted